src/test/compile-fail/rfc-2005-default-binding-mode/for.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+struct Foo {}
+
+// FIXME(tschottdorf): should this be a `ui` test?
```

Comment:
I'm generally in favor of moving to `ui` tests for everything (in fact I want to get rid of compile-fail eventually).